docs/governance/working-groups/delivery/charter.md

```diff
@@ -0,0 +1,50 @@
+# Delivery Working Group
+
+This [WG] defines and maintains the delivery process for the Common Cloud Controls project deliverables.
+
+## Mission
+
+The mission of the Delivery [WG] is to establish robust, scalable, and efficient frameworks for the governance and operation of all community [WG]s within the CCC.
+
+By designing clear guidelines and structures, this group ensures consistency and facilitates effective collaboration across the entire community. It acts as a central body to guide the formation and operation of other [WG]s, helping them align with the overarching goals of the CCC.
```

Comment:
This is from the community charter WG mission. Just to highlight that this should be changed/removed.

Reply:
It was updated in this commit: https://github.com/finos/common-cloud-controls/pull/167/commits/0019b73bad95b0b23a871087c0cf17c781a1f4bc

Let me know if you have any more concerns.